### PR TITLE
Smth grey txt files

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: SmthGrey/ReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+SmthGrey
+Smoothswing sound font for ProffieOS by Thexter.
+
+
+Modifications by Brian Conner 2022:
+Any additional sounds sourced from creative-commons licensed freesound.org.
+
+Hierarchy made to be FAT32 friendly.
+Added boot.wav.
```
Comment (profezzorn) [inline SmthGrey/ReadMe.txt:10]: These changes should be a part of the PR that submitted them, not batched together into one "txt file update".

**Comment by NoSloppy** [comment]
Updated.

File: SmthGrey/ReadMe.txt (hunk 1 above)
Comment (NoSloppy) [reply SmthGrey/ReadMe.txt:10]: I may have misinterpreted your request :

>do all the renames in one PR, then fix config files, and then any add/delete/modify separately

**Comment by NoSloppy** [comment]
closing and resubmitting as part of PR https://github.com/profezzorn/SoundFonts/pull/25

**Pull request closed without merging**